tcms/testplans/static/testplans/js/get.js

```diff
@@ -103,3 +114,38 @@ function getTestCaseExpandArea(row, testCase) {
         }
     });
 }
+
+function attachEvents() {
+    // update default tester
+    $('.js-test-case-menu-tester').click(function(ev) {
+        // todo this prompt need to be translated
+        var email_or_username = window.prompt('Please type new email or username');
+        if (!email_or_username) {
+            return;
+        }
+        const testCaseId = getCaseIdFromEvent(ev);
+
+        // default_tester can be update only if we pass an user id
+        jsonRPC('TestCase.update', [testCaseId, {'default_tester': email_or_username}], function(res) {
+            console.log(res);
+        });
+    });
+
+    // delete testcase from the plan
+    $('.js-test-case-menu-delete').click(function(ev) {
+        const testCaseId = getCaseIdFromEvent(ev);
+        jsonRPC('TestPlan.remove_case', [testPlanId, testCaseId], function(res) {
+           $("div").find(`[data-testcase-pk=${testCaseId}]`).remove();
+        });
+    });
+
+    // goto edit testcase page
+    $('.js-test-case-menu-edit').click(function(ev) {
+        const testCaseId = getCaseIdFromEvent(ev);
+        ev.target.href = `/case/${testCaseId}/edit`;
```

Comment:
you can set the target href in the calling context (or here) and don't need an onclick handler b/c the browser will handle it anyway, no ?